### DISCUSSION TITLE
Dungeon Quest

### DISCUSSION THREAD

**Discussion by strezzi96** [chromiecraft/chromiecraft#5136] [Suggestions]
Remember cata and dungeon quests at start of instance? Add those

**Comment by 55Honey** [comment]
Hello strezzi96,

thanks for your suggestion.
I'll keep my answer short and precise as your request, if you don't mind: No, such a thing won't happen anytime soon.

But maybe i can gouge your interest in the alpha version of the hardmode, which is currently available on the PTR. Feel free to try it in Magtheridon's and Gruul's Lair as well as the daily heroic dungeon.
